graylog2-server/src/main/java/org/graylog2/Configuration.java

```diff
@@ -79,7 +80,7 @@ public class Configuration extends CaConfiguration {
     private int outputFlushInterval = 1;
 
     @Parameter(value = "outputbuffer_processors", required = true, validators = PositiveIntegerValidator.class)
-    private int outputBufferProcessors = 3;
+    private int outputBufferProcessors = Math.round(Tools.availableProcessors() * 0.162f + 0.625f);
```

Comment:
Can you add a comment to explain the two magic numbers?

Reply:
What kind of explanation do you have in mind? I don't have a good one other than:
```
A linear function of available CPU cores, based on experiments. See https://github.com/Graylog2/graylog2-server/pull/17737.
```

Reply:
I've added some documetation